src/main/telemetry/crsf.c

```diff
@@ -238,6 +280,32 @@ typedef enum {
 static uint8_t crsfScheduleCount;
 static uint8_t crsfSchedule[CRSF_SCHEDULE_COUNT_MAX];
 
+void scheduleMspResponse() {
+    if (!mspReplyPending) {
+        mspReplyPending = true;
+    }
+}
+
+void crsfSendMspResponse(uint8_t *packet) 
+{
+    sbuf_t crsfPayloadBuf;
+    sbuf_t mspPayload;
+    sbuf_t *dst = &crsfPayloadBuf;
+    sbuf_t *msp = &mspPayload;
+
+    msp->ptr = packet;
+    msp->end = packet + CRSF_FRAME_TX_MSP_PAYLOAD_SIZE;
+
+    crsfInitializeFrame(dst);
+    sbufWriteU8(dst, CRSF_FRAME_TX_MSP_PAYLOAD_SIZE + CRSF_FRAME_LENGTH_EXT_TYPE_CRC);
+    sbufWriteU8(dst, CRSF_FRAMETYPE_MSP_RESP);
+    sbufWriteU8(dst, CRSF_ADDRESS_RADIO_TRANSMITTER);
+    sbufWriteU8(dst, CRSF_ADDRESS_BETAFLIGHT);
+    while (sbufBytesRemaining(msp)) {
```

Comment:
Actually, on second thoughts you can just do `sbufWriteData(dst, packet, CRSF_FRAME_TX_MSP_PAYLOAD_SIZE);` and avoid the need for creating the msp sbuf altogether.